eng/GenAPI.exclusions.txt

```diff
@@ -2,4 +2,19 @@
 T:Microsoft.AspNetCore.Components.RenderTree.RenderTreeFrame
 # Manually implemented - https://github.com/dotnet/arcade/issues/2066
 T:Microsoft.AspNetCore.Mvc.ApplicationModels.PageParameterModel
-T:Microsoft.AspNetCore.Mvc.ApplicationModels.PagePropertyModel
\ No newline at end of file
+T:Microsoft.AspNetCore.Mvc.ApplicationModels.PagePropertyModel
+# Manually implemented - https://github.com/aspnet/AspNetCore/issues/8825
+T:Microsoft.AspNetCore.Components.CascadingValue`1
+T:Microsoft.AspNetCore.Components.Forms.DataAnnotationsValidator
+T:Microsoft.AspNetCore.Components.Forms.EditForm
+T:Microsoft.AspNetCore.Components.Forms.InputBase`1
+T:Microsoft.AspNetCore.Components.Forms.InputDate`1
+T:Microsoft.AspNetCore.Components.Forms.InputNumber`1
+T:Microsoft.AspNetCore.Components.Forms.InputSelect`1
+T:Microsoft.AspNetCore.Components.Forms.InputText
+T:Microsoft.AspNetCore.Components.Forms.InputTextArea
+T:Microsoft.AspNetCore.Components.Forms.ValidationMessage`1
+T:Microsoft.AspNetCore.Components.Forms.ValidationSummary
+T:Microsoft.AspNetCore.Components.Layouts.LayoutDisplay
+T:Microsoft.AspNetCore.Components.Routing.NavLink
+T:Microsoft.AspNetCore.Components.Routing.Router
```

Comment:
All of our built-in components are now manual. I think this is the best thing to do since it's being spooky.

Reply:
Looks like we're missing `InputCheckbox` from here

Reply:
should be fixed now